webapi/src/test/scala/org/knora/webapi/slice/resources/repo/service/ResourcesRepoLiveSpec.scala

```diff
@@ -0,0 +1,283 @@
+              creationDate = valueCreationDate,
+              valueHasOrder = 1,
+            ),
+          ),
+          linkUpdates = Seq.empty,
+        )
+
+        val expected =
+          Update(s"""|
+                     |PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+                     |PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+                     |PREFIX owl: <http://www.w3.org/2002/07/owl#>
+                     |PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+                     |PREFIX knora-base: <http://www.knora.org/ontology/knora-base#>
+                     |
+                     |INSERT DATA {
+                     |    GRAPH <fooGraph> {
+                     |        <fooResource> rdf:type <fooClass> ;
+                     |            knora-base:isDeleted false ;
+                     |            knora-base:attachedToUser <fooUser> ;
+                     |            knora-base:attachedToProject <fooProject> ;
+                     |            rdfs:label \"\"\"fooLabel\"\"\" ;
+                     |            knora-base:hasPermissions \"fooPermissions\" ;
+                     |            knora-base:creationDate \"2024-01-01T10:00:00.673298Z\"^^xsd:dateTime .
+                     |
+                     |        
+                     |            # Value: fooValue
+                     |            # Property: fooProperty
+                     |
+                     |            
+                     |            <fooValue> rdf:type <http://api.knora.org/ontology/knora-api/v2#IntValue> ;
+                     |                knora-base:isDeleted false  ;
+                     |                knora-base:valueHasString \"\"\"42\"\"\" ;
+                     |                knora-base:valueHasUUID \"$uuidEncoded\" .
+                     |
+                     |
+                     |            
+                     |
+                     |                    <fooValue> knora-base:valueHasInteger 42 .
+                     |
+                     |                
+                     |
+                     |            
+                     |            
+                     |
+                     |            <fooValue> knora-base:attachedToUser <fooCreator> ;
+                     |                knora-base:hasPermissions \"fooValuePermissions\"^^xsd:string ;
+                     |                knora-base:valueHasOrder 1 ;
+                     |                knora-base:valueCreationDate \"2024-01-01T10:00:00.673298Z\"^^xsd:dateTime .
+                     |
+                     |
+                     |
+                     |            
+                     |            <fooResource> <fooProperty> <fooValue> .
+                     |        
+                     |
+                     |
+                     |        
+                     |    }
+                     |}
+                     |""".stripMargin)
```

Comment:
*thought*: I feel these `expected` Strings could need some formatting. 

On the other hand I assume the white space is necessary to make the test pass as the assertion depends on string equality which is not very ideal. Could we use some rdf4j parser for comparsion?

```pseudo code
val actualSparql: String = ???
val expectedSparql: String = ???
val someParser ???
assertTrue(someParser.parseOrThrow(actualSparql), omeParser.parseOrThrow(expectedSparql))
```

Reply:
I'll investigate options to improve this in the next PR